src/App.css

```diff
@@ -56,6 +56,15 @@ textarea {
   }
 }
 
+.TopBar.EmbeddedViewerVisible {
+  width: 100%;
+  /*position: absolute;*/
+  padding-left: 100px;
+  padding-right: 100px;
+  /*position: relative;*/
+  /*left: 153px*/
+}
+
```

Comment:
I think you should remove the commented out lines here if you're happy with the new implementation. Things can get pretty hard to read if you leave all the old code there! :smile:

Reply:
Oh yes, sorry, that's a pretty mashed up :sweat_smile: